$OpenBSD: patch-fingerprint_ml,v 1.1 2005/05/30 19:27:45 sturm Exp $
Post-release developer patch.
o Fix the file open in text mode instead of binary.
  See http://groups.yahoo.com/group/unison-users/message/3184
  and http://lists.seas.upenn.edu/pipermail/unison-hackers/2005-May/000089.html
--- fingerprint.ml.orig	Mon Sep  6 15:15:47 2004
+++ fingerprint.ml	Mon May 23 16:30:14 2005
@@ -22,7 +22,7 @@ let subfile path offset len =
   Util.convertUnixErrorsToTransient
     "digesting subfile"
     (fun () ->
-       let inch = open_in path in
+       let inch = open_in_bin path in
        LargeFile.seek_in inch offset;
        begin try
          let res = Digest.channel inch (Uutil.Filesize.toInt len) in
